src/js/letters/reducers/index.js

```diff
@@ -5,7 +5,7 @@ import _ from 'lodash/fp';
 const initialState = {
   letters: [],
   destination: {},
-  lettersAvailable: false,
+  lettersAvailability: 'awaitingResponse',
```

Comment:
I think we should define constants for these 3 state values. Not sure if in this file or elsewhere, but then import them so all files are referring to the same constants. All-caps would also be good for readability.